packages/create-remix/templates/fly-stack/README.md

```diff
@@ -0,0 +1,85 @@
+# Remix Fly Stack
+
+- [Remix Docs](https://remix.run/docs)
+
+## Fly Setup
+
+1. [Install Fly](https://fly.io/docs/getting-started/installing-flyctl/)
+
+2. Sign up and log in to Fly
+
+```sh
+$ flyctl auth signup
+```
+
+3. You'll need to create a .env file in your project root for your SESSION_SECRET
+
+```sh
+$ cp .env.example .env
+$ echo SESSION_SECRET=$(openssl rand -hex 32) >> .env
+```
+
+## The Database
+
+In development, it's better to use a local database, The easiest way to do this is using [Docker][docker]. To start your postgres database, run the following command:
+
+```sh
+$ docker-compose up
+```
+
+That may take a moment to start up as it needs to get the postgres image from the Docker registry, after it's done, you'll need to migrate your database
+
+```sh
+$ prisma migrate deploy
```

Comment:
```suggestion
npx prisma migrate deploy
```

Reply:
This command breaks because we haven't run `npm install` yet. Let's add that step somewhere before this one.